### ISSUE TITLE
problem of login dialog box on windows

### ISSUE THREAD

**Issue by LorenzoTa** [choroba/pm-cb#48]
hello, 
when login dialog box spawns it seems not selectable on Windows: it onyl appears if I hover the mouse on the chat on the toolbar, but is not selectable
I'll investigate further as I have time

**Comment by choroba** [comment]
The startup login window is OK, the problem appears when the client gets logged out and displays the window to relog in.

**Comment by LorenzoTa** [comment]
yes, sorry if I was not clear enough in my previous description.
anyway I noticed strange behaviors in tk on win10: the OS is used to group windows in the application bar and I noticed tk application not grouped with the command prompt window where the tk application was created, but with another one (that probably has the same name under the registry under HKCU\Console\full_path_to_perl_exe_with_underscores_as_separator ) This seems to work correctly on win7.
At the moment I'm monitoring it on win7 to see what happens if a logout will happen
